Ark.Tools.FtpClient.FtpProxy/FtpClientProxy.cs

```diff
@@ -61,7 +63,9 @@ internal FtpClientProxy(IFtpClientProxyConfig config, IFlurlClientFactory client
         [Obsolete("Use the constructor with FtpConfig", false)]
         internal FtpClientProxy(IFtpClientProxyConfig config, IFlurlClientFactory client, TokenProvider tokenProvider, Uri uri, NetworkCredential credentials)
         {
-            _init(config, null, uri, credentials, null);
+            using var ftpConfig = new FtpConfig(uri, credentials);

```

Comment:
add a comment as this is 'fine' to dispose the FtpConfig because has no 'Certificate' in this particular case.